reference/services/buildbot.rst

```diff
@@ -0,0 +1,158 @@
+Buildbot
+========
+
+Short description
+-----------------
+
+`Buildbot <https://github.com/buildbot/buildbot>`__ is a Python-based
+continuous integration testing framework that we use to run tests,
+manage stable branches for Launchpad. We use `Buildbot
+0.8.12 <https://docs.buildbot.net/0.8.12/>`__ in production currently.
+You can access our hosted instance at http://lpbuildbot.canonical.com
+
+Detailed description
+--------------------
+
+There are the following main components in the Launchpad Buildbot
+setup. 
+
+1. `lpbuildbot <https://code.launchpad.net/~canonical-launchpad-branches/lpbuildbot/>`__:
+   This contains the \`master.cfg\` file and the related Python code
```

Comment:
Is this on purpose to escape the \`? Below we go back to \`\`master.cfg\`\`
I'd just use ` without escaping. There are a few others below